superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/FilterScope/ScopingTree.tsx

```diff
@@ -29,29 +29,36 @@ type ScopingTreeProps = {
   updateFormValues: (values: any) => void;
   formScope?: Scope;
   initialScope: Scope;
+  chartId?: number;
```

Comment:
Should we rather make this `excludedChartIds?: number[]`? Now it's unclear what the component needs the `chartId` for, and it might well be that we want to exclude some other charts other than the caller.